templates/index_en.html

```diff
@@ -7,11 +7,16 @@
             <div class="flex justify-between items-center">
                 <a href="/" class="text-white text-2xl md:text-3xl font-extrabold">ME25</a>
                 <div class="flex items-center space-x-4">
-                    <a href="https://www.linkedin.com/company/munich-embedded-conference" target="_blank" title="Follow our LinkedIn Page" class="px-4 text-white hover:text-blue-200">
+                    <a href="https://www.linkedin.com/company/munich-embedded-conference" target="_blank" title="Unserer LinkedIn Seite folgen" class="px-1 text-white hover:text-blue-200">
```

Comment:
```suggestion
                    <a href="https://www.linkedin.com/company/munich-embedded-conference" target="_blank" title="Follow our LinkedIn Page" class="px-1 text-white hover:text-blue-200">
```
Oops, too much copy&paste